code/game/machinery/atmo_control.dm

```diff
@@ -545,6 +546,11 @@
 
 	add_fingerprint(usr)
 
+	if(href_list["in_set_rate"])
+		var/response=input(usr,"Set new flow rate, in L/s. \[0-[200]\]") as num
```

Comment:
Nit: please keep a space at each side of `=` in assignment, and after `,` in argument list.

Reply:
I copy/pasted the bulk of that line from the output pressure equivalent, which is where that jankiness came from. I have now fixed it in both.